### PR TITLE
Wdp 148

### PR DESCRIPTION
**Pull request by gonczor**
image
image
https://kindergarten.stxnext.pl/browse/WDP-148

### CODE REVIEW

File: runtime/data/test_data/test_data.csv [hunk 1]
```diff
@@ -6,4 +6,15 @@
...
+13,2013-09-09,09:18:53,17:25:44
+13,2013-09-10,09:31:57,16:12:10
+13,2013-09-11,09:06:09,17:36:09
+13,2013-09-12,09:14:47,17:30:59
+14,2011-06-01,12:56:33,17:41:34
+14,2011-06-02,10:02:15,17:18:22
+14,2011-06-03,09:33:23,23:58:24
+14,2011-06-04,00:03:24,02:58:24
+14,2011-06-06,08:33:27,17:07:19
+
```
Comment (piotrgrundas) [inline runtime/data/test_data/test_data.csv]: remove

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

File: src/presence_analyzer/tests.py [hunk 2]
```diff
@@ -76,6 +79,23 @@ def test_api_users(self):
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'Maciej Z.'})
         self.assertDictEqual(data[1], {u'user_id': 11, u'name': u'User 11'})
 
+    def test_api_users_is_sorted(self):
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: users is sorted?

File: src/presence_analyzer/views.py [hunk 3]
```diff
@@ -23,6 +24,7 @@
 from .main import app
 
 log = logging.getLogger(__name__)  # pylint: disable=invalid-name
+locale.setlocale(locale.LC_ALL, '')
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: why LC_ALL?

File: src/presence_analyzer/tests.py [hunk 4]
```diff
@@ -34,6 +35,8 @@
     os.path.dirname(__file__), '..', '..', 'runtime', 'data', 'test_data', 'test_users.xml'
 )
 
+locale.setlocale(locale.LC_CTYPE, ('en', 'utf-8'))
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: why ctype?

File: src/presence_analyzer/tests.py [hunk 5]
```diff
@@ -65,16 +68,19 @@ def test_mainpage(self):
...
         self.assertEqual(resp.content_type, 'application/json')
         data = json.loads(resp.data)
-        self.assertEqual(len(data), 2)
-        self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'Maciej Z.'})
-        self.assertDictEqual(data[1], {u'user_id': 11, u'name': u'User 11'})
+        self.assertEqual(len(data), 4)
+        self.assertDictEqual(data[0], {u'user_id': 11, u'name': u'Maciej D.'})
+        self.assertDictEqual(data[1], {u'user_id': 10, u'name': u'Maciej Z.'})
+        self.assertDictEqual(data[2], {u'user_id': 13, u'name': u'User 13'})
+        self.assertDictEqual(data[3], {u'user_id': 130, u'name': u'User 130'})
```
Comment (piotrgrundas) [inline src/presence_analyzer/tests.py]: use unicode literals

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

File: src/presence_analyzer/tests.py (hunk 4 above)
Comment (gonczor) [reply src/presence_analyzer/tests.py]: Should I use LC_COLLATE instead? I wasn't sure which to choose.

File: src/presence_analyzer/views.py [hunk 6]
```diff
@@ -5,6 +5,7 @@
 
 import calendar
 import logging
+import locale
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py:8]: sort imports

**Pull request merged**